tests/cases/standalone/common/tql-explain-analyze/analyze.result

```diff
@@ -39,7 +39,7 @@ TQL ANALYZE (0, 10, '1s', '2s') test;
 +-+-+
 | plan_type_| plan_|
 +-+-+
-| Plan with Metrics | PromInstantManipulateExec: range=[0..10000], lookback=[300000], interval=[1000], time index=[j], REDACTED
+| Plan with Metrics | PromInstantManipulateExec: range=[0..10000], lookback=[2000], interval=[1000], time index=[j], REDACTED
```

Comment:
@waynexia we even support parse lookback but not convey it to the execution 🤣.

Reply:
Sorry, not sure I got your point here.
Once the lookback parameter is passed to TQL query it’s got applied. 
Did you mean that there’s no way at the moment to pass it down to that level?

Reply:
@etolbakov IIUC `2s` in `TQL ANALYZE (0, 10, '1s', '2s') test` is the lookback arg. We previous pass this arg but doesn't respect it in the execution?

It's not quite normal the `.result` files changed without `.sql` file changes.

Reply:
Ahhh… I see where the confusion comes from.
Let me try to explain.
In my previous pr (related to time function support https://github.com/GreptimeTeam/greptimedb/pull/3620 ) I added the lookback parameter just as proof of support. It didn’t get applied to the result.

With the current scope of changes it started taking an effect on the result.
That’s why only the result files have been changed (in other words actualized)

Reply:
Seems https://github.com/GreptimeTeam/greptimedb/pull/3507.